src/pages/Survey/Survey.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+
+// CONPONENTS
+import Hero from './HeroSection/Hero';
+import InfoBox from './InfoBox/InfoBox';
+import SurveyBox from './SurveyBox/SurveyBox';
+
+// STYLES
+import styled from 'styled-components';
+
+import { BASE_URL } from '../../config';
+
+function Survey() {
+  const [pageInfo, setPageInfo] = useState([]);
+
+  useEffect(() => {
+    getInfo();
+  }, []);
+
+  const getInfo = () => {
+    fetch(`${BASE_URL}/services/2`)
+      .then(res => res.json())
+      .then(res => setPageInfo(res.results[0]));
+  };
```

Comment:
```suggestion
  useEffect(() => {
    const getInfo = () => {
      fetch(`${BASE_URL}/services/2`)
        .then(res => res.json())
        .then(res => setPageInfo(res.results[0]));
    };
  
    getInfo();
  }, []);
```
지금 코드 상으로도 크게 문제는 없지만 지금 방식으로는 의존성 배열을 속여 쓰게 되는 부분이 있습니다
정확히는 getInfo도 의존성 배열에 넣어줘야 합니다. 하지만 그렇게 되면 매 렌더마다 getInfo 함수의 인스턴스가 생성되고, 무한렌더가 될거라서 useCallback으로 해당 함수를 감싸라고 워닝이 뜰건데 그럴 필요 없이 애초에 useEffect 안에서 사용하는 함수를 그냥 같이 포함시켜주면 문제가 되지 않습니다.

지금 수정하실 필요는 없고 나중에 공부해보세요!
https://rinae.dev/posts/a-complete-guide-to-useeffect-ko#%ED%95%98%EC%A7%80%EB%A7%8C-%EC%A0%80%EB%8A%94-%EC%9D%B4-%ED%95%A8%EC%88%98%EB%A5%BC-%EC%9D%B4%ED%8E%99%ED%8A%B8-%EC%95%88%EC%97%90-%EB%84%A3%EC%9D%84-%EC%88%98-%EC%97%86%EC%96%B4%EC%9A%94

Reply:
네 공부하겠습니다 감사합니다 ㅎㅎㅎ 저 이런거 완전 좋아해요